File: IsometricShooter/Scripts/JS/Configuration/PlayerConf.js
#pragma strict

var nickname : String = "Player";  //Nickname of player
var speedHorizontal : float = 5.0; //Speed at horizontal movement
var speedVertical : float = 5.0; //Speed at vertical movement
var speedJump : float = 5.0; //Speed at jump movement
var onHitEffect : GameObject; //Effect on hit by projectile
var onHitEffectType : Effect; //Type of effect on hit by projectile
var onDieEffect : GameObject; //Effect on die by projectile
var onDieEffectType : Effect; //Type of effect on die by projectile
var sfxHit : AudioClip; //Audio to be played in case of bullet hit
var sfxDie : AudioClip; //Audio to be played in case of death
var sfxScream : AudioClip; //Audio to be played in case of scream

File: IsometricShooter/Scripts/CS/Configuration/PlayerConf.cs
using UnityEngine;
using System.Collections;

public class PlayerConf : MonoBehaviour {

	public string nickname = "Player";  //Nickname of player
	public float speedHorizontal = 5.0f; //Speed at horizontal movement
	public float speedVertical = 5.0f; //Speed at vertical movement
	public float speedJump = 5.0f; //Speed at jump movement
	public GameObject onHitEffect; //Effect on hit by projectile
	public Effect onHitEffectType; //Type of effect on hit by projectile
	public GameObject onDieEffect; //Effect on die by projectile
	public Effect onDieEffectType; //Type of effect on die by projectile
	public AudioClip sfxHit; //Audio to be played in case of bullet hit
	public AudioClip sfxDie; //Audio to be played in case of death
	public AudioClip sfxScream; //Audio to be played in case of scream
	
}
